primefaces-integration-tests/src/test/java/org/primefaces/integrationtests/selectonemenu/SelectOneMenu001Test.java

```diff
@@ -131,6 +132,24 @@ public void testEnablePanel(Page page) {
         assertConfiguration(selectOneMenu.getWidgetConfiguration());
     }
 
+    @Test
+    @Order(5)
+    @DisplayName("SelectOneMenu: selectValue via JavaScript")
+    public void testJsSelectValue(Page page) {
+        // Arrange
+        SelectOneMenu selectOneMenu = page.selectOneMenu;
+        Assertions.assertEquals("Lewis", selectOneMenu.getSelectedLabel());
+
+        // Act
+        PrimeSelenium.executeScript("PF('selectonemenu').selectValue('Max');");
+        selectOneMenu.select("Max");
```

Comment:
Why do you also call `select` after calling `selectValue` ?

Reply:
Uuups. copying-issue.